pkg/v13patch1/chartconfig/desired.go

```diff
@@ -0,0 +1,128 @@
+package chartconfig
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/errors/guest"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/pkg/v13patch1/key"
+)
+
+const (
+	chartConfigAPIVersion           = "core.giantswarm.io"
+	chartConfigKind                 = "ChartConfig"
+	chartConfigVersionBundleVersion = "0.4.0"
```

Comment:
That looks pretty fishy. Why should we hide it here? Why do we need it at all? I fear this gets forgotten and tends to be not fully understood because there is this magic string hidden somewhere and will just cause problems in the future.